jsdoc.json

```diff
@@ -0,0 +1,20 @@
+{
+  "source": {
+    "exclude": [
+      "__snapshots__",
+      "build",
+      "coverage",
+      "logo",
+      "node_modules",
+      "private",
+      "public"
+    ]
+  },
+  "plugins": ["plugins/markdown"],
+  "opts": {
```

Comment:
We could also specify a theme here too, if we wanted but I've gone with the default for now